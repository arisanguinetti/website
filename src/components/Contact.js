import React, { Component } from "react";

class Contact extends Component {
  render() {
    return (
      <section id="contact" className="section">
        <div className="container section-wrapper">
          <div className="section-content">
            <div className="row">
              <div className="col-md-12 text-center">
                <h2 className="section-title">Contact Me</h2>

                <p className="section-subtitle">
                  Have a project you'd like to discuss?
                </p>

                <span className="divider center" />
              </div>
            </div>

            <div className="row">
              <div className="col-md-12">
                <div className="contact-details text-center">
                  <div className="speech-bubble">
                    <i className="icon-Speach-Bubble11" />

                    <h4 className="text-grey">
                      Say
                      <br />
                      Hello
                    </h4>
                  </div>

                  <h2>
                    Ariel <strong>Sanguinetti</strong>
                  </h2>

                  <h4 className="text-grey">
                    <a href="mailto:hello@arielsanguinetti.com.ar">
                      hello@arielsanguinetti.com.ar
                    </a>
                  </h4>

                  <span className="divider center" />
                </div>
              </div>
            </div>
          </div>
        </div>
      </section>
    );
  }
}

export default Contact;
