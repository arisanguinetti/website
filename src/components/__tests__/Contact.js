import React from "react";
import { shallow } from "enzyme";
import Contact from "../Contact";

describe("Contact", () => {
  describe("Rendering", () => {
    it("should match to snapshot", () => {
      process.env.REACT_APP_ARIEL_EMAIL = "john@example.com";
      const component = shallow(<Contact />);
      expect(component).toMatchSnapshot();
    });
  });
});
