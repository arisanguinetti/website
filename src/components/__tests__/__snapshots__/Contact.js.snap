// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Rendering should match to snapshot 1`] = `
<section
  className="section"
  id="contact"
>
  <div
    className="container section-wrapper"
  >
    <div
      className="section-content"
    >
      <div
        className="row"
      >
        <div
          className="col-md-12 text-center"
        >
          <h2
            className="section-title"
          >
            Contact Me
          </h2>
          <p
            className="section-subtitle"
          >
            Have a project you'd like to discuss?
          </p>
          <span
            className="divider center"
          />
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="col-md-12"
        >
          <div
            className="contact-details text-center"
          >
            <div
              className="speech-bubble"
            >
              <i
                className="icon-Speach-Bubble11"
              />
              <h4
                className="text-grey"
              >
                Say
                <br />
                Hello
              </h4>
            </div>
            <h2>
              Ariel 
              <strong>
                Sanguinetti
              </strong>
            </h2>
            <h4
              className="text-grey"
            >
              <a
                href="mailto:hello@arielsanguinetti.com.ar"
              >
                hello@arielsanguinetti.com.ar
              </a>
            </h4>
            <span
              className="divider center"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</section>
`;
